src/main/routes/task-list.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express';
+import { SubTaskListBuilder } from '../app/case/util/subTaskListBuilder';
+
+const router = express.Router();
+
+router.get('/task-list', (req, res) => {
```

Comment:
you probably seen it already, I plan to put all paths in paths file from which a11y tests pick up automatically. OK for now but this has to be changed soon.

Reply:
Yes I seen and I like that change 👍